examples/games/game_menu.rs

```diff
@@ -45,10 +45,62 @@ fn setup(mut commands: Commands) {
     commands.spawn(Camera2dBundle::default());
 }
 
+// This outermost node encompasses the entire Window (100% width and height)
+// and centers all content vertically (align_items) and horizontally (justify_content)
+fn outer_node(
```

Comment:
This is a pattern common to every screen of this example, and was previously repeated verbatim in 6 different places